dialogue/TownNPCs/Mocha/InitialInteractions/CardinalPistachio/CardinalPistachio_Mocha_First2.json

```diff
@@ -21,7 +21,7 @@
     ],
     "pl": [
       "Kurcze, więc udało mu się uczec, ha... Miasteczko w nieładzie oznacza złe wieści dla moich rezerwacjach.",
```

Comment:
Use "uciec" instead of "uczec", and I also think "co..." will be better than "ha...". Also a slight grammatical error, change "rezerwacjach" to "rezerwacji".

Reply:
Oops, I'm sure I used "uciec" elsewhere, seems to be a slip up. Cases is something I do muddle up sometimes, but thanks.

Reply:
That's okay, mistakes happen